lib/cluster.js

```diff
@@ -25,7 +27,22 @@ class AggregatorRegistry extends Registry {
 	constructor() {
 		super();
 		addListeners();
-	}
+    this.aliveWorkers = new Set();
+    let that = this;
+		cluster.on('message', function(worker, message) {
+			if (arguments.length === 2) {
+				// pre-Node.js v6.0
+				message = worker;
+				worker = undefined;
+			}
+      if (message.type === REG_METRICS_WORKER) {
+        let workerId = message.workerId;
+        console.log("metrics registering workerId", workerId);
+        that.aliveWorkers.add(workerId);
+        console.log(that.aliveWorkers);
+      }
+    });
```

Comment:
Instead of adding another `cluster.on("message")` listener, can you do this in `addListeners`?

Reply:
To do that I might have to move the aliveListeners to be a module-level variable. Would that be ok?